js/src/components/google-account-card/google-account-card.js

```diff
@@ -8,15 +8,22 @@ import AuthorizeGoogleAccountCard from './authorize-google-account-card';
 import ConnectedGoogleAccountCard from './connected-google-account-card';
 
 export default function GoogleAccountCard( { disabled = false } ) {
-	const { google, hasFinishedResolution } = useGoogleAccount();
+	const { google, scope, hasFinishedResolution } = useGoogleAccount();
 
 	if ( ! hasFinishedResolution ) {
 		return <AccountCard appearance={ {} } description={ <AppSpinner /> } />;
 	}
 
-	if ( google?.active === 'yes' ) {
+	if ( google?.active === 'yes' && scope.gmcRequired ) {
 		return <ConnectedGoogleAccountCard googleAccount={ google } />;
 	}
 
-	return <AuthorizeGoogleAccountCard disabled={ disabled } />;
+	const additionalScopeEmail = scope.gmcRequired ? undefined : google.email;
```

Comment:
🚧  When jetpack account is disconnected, there is an error thrown: `TypeError: Cannot read properties of undefined (reading 'email')`.

This seems to fix it:

```suggestion
	const additionalScopeEmail = scope.gmcRequired ? undefined : google?.email;
```

Reply:
Oops! Thanks for catching this! 👍